content/shared/js/highlight.pack.js

```diff
@@ -1,2 +1,337 @@
\ No newline at end of file
+/*!
+  Highlight.js v11.9.0 (git: b7ec4bfafc)
+  (c) 2006-2023 undefined and other contributors
```

Comment:
Looks like the license export might have glitched

Reply:
@nschonni thanks for catching this. I just downloaded the file for consumption. Is there a good way to fix this?

Reply:
Not sure, maybe it was a temporary glitch on the site generator, assuming that's how you grabbed it

Reply:
Yep confirming I'm also getting `undefined` from the exporter at https://highlightjs.org/download. We could make an assumption about that name based on the [repository's LICENSE](https://github.com/highlightjs/highlight.js/blob/main/LICENSE), though we shouldn't add include it.

Seems like something we can make an issue about on their repository and update this file when that's fixed.